.circleci/config.yml

```diff
@@ -1105,6 +1106,21 @@ jobs:
     steps:
       - run: echo Done
 
+  fpp-verify:
+    docker:
+      - image: cimg/go:1.19
+    steps:
+      - checkout
+      - run:
+          name: verify-goerli
+          command: |
+            make verify-goerli
+          working_directory: op-program
+      - slack/notify:
+          channel: C03N11M0BBN
+          event: fail
```

Comment:
Is there a way to know if the job ran at all? Since it runs pretty infrequently, maybe it's fine to get notified all the time.

Reply:
We can check in the CircleCI dashboard. We used to have the hive tests posting to slack on every run and it just created a lot of noise that hid the failures.

Reply:
I see. It's better this way then.